src/ripple/app/tx/applySteps.h

```diff
@@ -86,35 +86,24 @@ struct PreclaimResult
 
     /// Intermediate transaction result
     TER const ter;
-    /// Transaction-specific base fee
-    std::uint64_t const baseFee;
     /// Success flag - whether the transaction is likely to
     /// claim a fee
     bool const likelyToClaimFee;
 
     /// Constructor
     template<class Context>
     PreclaimResult(Context const& ctx_,
-        TER ter_, std::uint64_t const& baseFee_)
+        TER ter_)
```

Comment:
If you get a chance, remove this newline please.

Reply:
Done.